frontend/src/component/operator-wizard/OperatorWizard.js

```diff
@@ -52,9 +53,9 @@ export default class OperatorWizard extends Component {
   };
 
   onDeploy = () => {
-    if (!this.validateForm()) {
-      return;
-    }
+    // if (!this.validateForm()) {
+    //   return;
+    // }
     const result = this.createResultYaml();
     console.log(result);
```

Comment:
@swati-kale Do we really need this? Or it is for testing?

Reply:
yes log was for testing, but can be removed.